apps/halo/2.20.11/docker-compose.yml

```diff
@@ -1,6 +1,6 @@
 services:
   halo:
-    image: halohub/halo-pro:2.20.11
+    image: halohub/halo-pro:2.20.12
     container_name: ${CONTAINER_NAME}
     restart: always
     networks:
```

Comment:
No issues found with the provided code snippet. The patch updates the Halo Hub image version from `2.20.11` to `2.20.12`, which is typical practice for updating software installations. There are no syntax errors or other problems in this change.